es6-classes/square.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(area, perimeter, width) {
+    super(area, perimeter);
```

Comment:
The only parameter for this `constructor` should be `width`, and utilizing your `width` you should be calculating what you'd want for your `area` & `perimeter` and pass those into `super()`